packages/misc/src/components/LinkButton/types.ts

```diff
@@ -0,0 +1,13 @@
+import { CSSProperties } from 'react';
+import { ButtonIconVariant } from 'react-rainbow-components/components/types';
+import { LinkProps } from 'react-router-dom';
+
+export interface ButtonProps {
+    className: string;
+    style: CSSProperties;
+    variant?: ButtonIconVariant;
+    shaded?: boolean;
+    size?: 'small' | 'medium' | 'large';
+}
+
+export type LinkButtonProps = React.PropsWithChildren<ButtonProps & LinkProps>;
```

Comment:
using LinkProps means that the component will accept all the Link props and we will only use `to` for now